heron/stmgr/src/cpp/manager/stmgr-server.cpp

```diff
@@ -162,7 +173,18 @@ sp_string StMgrServer::MakeBackPressureCompIdMetricName(const sp_string& instanc
 }
 
 sp_string StMgrServer::MakeQueueSizeCompIdMetricName(const sp_string& instanceid) {
-  return METRIC_QUEUE_SIZE_TO_INSTANCE_COMPID + instanceid;
+  return CONNECTION_BUFFER_BY_INSTANCEID + instanceid;
+}
+
+void StMgrServer::UpdateQueueMetrics(EventLoop::Status) {
+  for (auto itr = active_instances_.begin(); itr != active_instances_.end(); ++itr) {
+    sp_int32 task_id = itr->second;
+    const sp_string& instance_id = instance_info_[task_id]->instance_->instance_id();
+    sp_int32 bytes = itr->first->getOutstandingBytes();
+    connection_buffer_metric_map_[instance_id]->scope("bytes")->record(bytes);
```

Comment:
after long run, the mean metric becomes obtuse. 
for example, numerator_=10*1000, denominator_=1000, the mean is 10. 
then, 20 value pulse 100 times only lead to 0.9 increase on the mean. 
numerator_=10*1000+20*100, denominator_=1000+100, the mean is 10.9. 
is it the intention?